{{cookiecutter.project_slug}}/Makefile

```diff
@@ -21,7 +21,7 @@ export DEFAULT_VENV_NAME
 
 # Options for all calls to up-do-date pip (i.e., AFTER `pip install -U pip`)
 # Example: `--use-feature=2020-resolver` before the new resolver became the default
-PIP_OPTS = --use-feature=in-tree-build
+PIP_OPTS = # --use-feature=in-tree-build is now default in pip
```

Comment:
Very minor: I'd personally put another `#` before `is now...` so removing the main `#` wouldn't break the code. Also, the flag could also just be removed instead of being commented, given it's not necessary anymore (the same happened to `2020-resolver`).